### DISCUSSION TITLE
How to export Data Objects Importer configurations?

### DISCUSSION THREAD

**Discussion by ghost** [pimcore/pimcore#9849] [Q&A / Support] [answered]
Just as it is possible to export classes, is it possible to export Data Objects Importer configurations?
If so, how to proceed?

**Comment by sissonen** [comment, marked as answer]
The datahub configurations (both graphQL and Data Objects Importer) are saved in var/config/datahub-configurations.php.

If you move the config file to another environment you'll need to run "bin/console datahub:graphql:rebuild-definitions" for GraphQL changes (not sure about data importer).

**Reply by fashxp** [reply]
an export feature would be quite a nice addition I would say.
